index.html

```diff
@@ -387,10 +395,10 @@ <h3>Skills</h3>
           </ul>
         </div>
       </section>
-    </section>
+    </div>
 
     <footer>
-      <section class="form-section">
+      <section class="form-section" id="contact">
```

Comment:
- [ ] [OPTIONAL] I have noticed that in mobile size the footer section looks broken. The about myself part is going out of flow, and the form is too small, I have recorded a video suggesting solutions to fix this issues and making your page look even better! 😉 

https://user-images.githubusercontent.com/77038610/146581962-8ee14422-ba6f-4161-8bc3-e37a2696cea5.mp4